Nvidia_GPU_Temperature.py

```diff
@@ -14,19 +14,20 @@
 
 while True:
 
-    output = subprocess.check_output(["C:\\Program Files\\NVIDIA Corporation\\NVSMI\\nvidia-smi.exe", "-a"], shell=True)
+    output = str(subprocess.check_output(["C:\\Program Files\\NVIDIA Corporation\\NVSMI\\nvidia-smi.exe", "-a"], shell=True))
     #os.popen('C:\\Program Files\NVIDIA Corporation\NVSMI\nvidia-smi.exe')
     #output=os.popen("C:\\Program Files\\NVIDIA Corporation\\NVSMI\\nvidia-smi.exe").read()
-    
+
     #print("====" + str(output) + "=====")
     temp = re.search("GPU Current.*",output).group()[30:33]
-    temp_baseline = 60
+
+    temp_baseline = 55  # configure to you base line temp here
```

Comment:
accidental change?

Reply:
intentional. Please note the comment. Different GPUs have different baseline temps.

Reply:
the title of this pull request says "updated to work in both python 2 and 3", thus changes of this kind are ... unexpected.